specs/DragStarting.md

```diff
@@ -0,0 +1,196 @@
+DragStarting
+===
+
+# Background
+The WebView2 team has been asked to provide a way to override the default drag drop behavior when running in visual hosting mode. This event allows you to know when a drag is initiated in WebView2 and provides the state necessary to override the default WebView2 drag operation with your own logic.
```

Comment:
Did you also have an interop COM interface for this API?

Reply:
Yes. Are interop interfaces included in review? If so I'll add it.

Reply:
Added the interop interface